recipes/recipes_emscripten/llvm/patches/fix_clang_keywords.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/clang/lib/Parse/ParseTentative.cpp b/clang/lib/Parse/ParseTentative.cpp
+index ff27ef70944a..27cdf2f90f51 100644
+--- a/clang/lib/Parse/ParseTentative.cpp
++++ b/clang/lib/Parse/ParseTentative.cpp
+@@ -1504,6 +1504,7 @@ Parser::isCXXDeclarationSpecifier(ImplicitTypenameContext AllowImplicitTypename,
+   case tok::kw_inline:
+   case tok::kw_virtual:
+   case tok::kw_explicit:
++  case tok::kw__Noreturn:
+ 
+     // Modules
+   case tok::kw___module_private__:
+@@ -1558,6 +1559,7 @@ Parser::isCXXDeclarationSpecifier(ImplicitTypenameContext AllowImplicitTypename,
+     // GNU
+   case tok::kw_restrict:
+   case tok::kw__Complex:
++  case tok::kw__Imaginary:
+   case tok::kw___attribute:
+   case tok::kw___auto_type:
+     return TPResult::True;
```

Comment:
Upstream patch has been merged and we can get rid of this by the next release (12th June)

Fixes this issue https://github.com/compiler-research/xeus-cpp/issues/320